### ISSUE TITLE
dataviz - type pie - plot preview

### ISSUE THREAD

**Issue by josemvm** [3liz/lizmap-plugin#478]
## Description

for the pie type dataviz, the plot preview doens't take in account the layout code:
`{"colorway": ["#ffffff", "#57a163", "#e2d251", "#086fa1", "#bc4a3c", "#856e41", "#000000", "#ffffff", "#8a9a97"]}`

lizmap image:
![imagem](https://user-images.githubusercontent.com/10053874/221549341-0b712b41-7c9e-4cbe-910c-969545795882.png)

preview image:
![imagem](https://user-images.githubusercontent.com/10053874/221548870-ffaf74bd-ed6b-485b-8a1d-66836356edb8.png)

## Context
* QGIS version: 3.22.16
* Lizmap version: 3.6.2 pre
* OS: Ubuntu 20

**Comment by Gustry** [comment]
I'm not sure if it's a bug on Lizmap Web Client side or here.
The plugin is taking the result from the LWC server only.

Does the layout works for other charts ?

**Comment by josemvm** [comment]
for the scatter type dataviz, the plot preview doens't show de line

lizmap image:
![imagem](https://github.com/3liz/lizmap-plugin/assets/10053874/ebedf4e8-5d45-46f7-b95b-b9b231525bf4)

preview image:
![imagem](https://github.com/3liz/lizmap-plugin/assets/10053874/f67556b8-a3ea-4d04-bcfe-53bca8c901a7)

thanks

**Comment by mdouchin** [comment]
Indeed, the code does not take into account the [additional JSON layout ](https://docs.lizmap.com/current/fr/publish/lizmap_plugin/dataviz.html#json-layout) given by the user

We could add in the code just before [this line](https://github.com/3liz/lizmap-plugin/blob/master/lizmap/resources/html/dataviz.html#L14)

```javascript
// Apply the JSON user layout
Plotly.relayout("lizmap-plot", {plot_user_layout});
```
